src/pandablocks_ioc/_pvi.py

```diff
@@ -256,4 +255,19 @@ def create_pvi_records(record_prefix: str):
 
             for device in devices:
                 bobfile_path = Pvi._screens_dir / Path(f"{device.label}.bob")
+
+                if device.label in [
+                    group.name.split(":")[0]
+                    for group in _positions_table_group.children
+                ]:
+                    # The device is in the positions table
+                    device_ref = DeviceRef(
+                        "AllPostionCaptureParameters",
+                        "CAPTURE",
+                        "PandA_POSITIONS_TABLE",
+                    )
+                    for group in device.children:
+                        if group.name == "CAPTURE":
+                            cast(Group, group).children.append(device_ref)
+
                 formatter.format(device, record_prefix + ":", bobfile_path)
```

Comment:
I'm not clear what this code is doing...

Reply:
I'm hoping it's a lot clearer now!